lib.php

```diff
@@ -11,14 +11,17 @@
 
 require_once(get_config('docroot') . 'module/objectfs/objectfslib.php');
 require_once(get_config('docroot') . 'module/objectfs/classes/client/s3_client.php');
```

Comment:
These need to become dynamic requires.

Reply:
Can you provide me an example of what you mean by dynamic require?

Reply:
Like this: https://github.com/catalyst/moodle-tool_objectfs/blob/master/lib.php#L120 but with a require built in - moodle doesn't need to require cause it has appropriate auto loading